src/boot/api.js

```diff
@@ -29,6 +29,9 @@ const signTransaction = async function (actions) {
         expireSeconds: 30
       })
       transactionId = result.transaction_id
+    } else if (typeof window.LightWalletChannel === 'object') {
+      const firstAction = actions[0]
+      transactionId = await this.$lightWallet.sendTransaction(firstAction)
```

Comment:
You can't only sign the first action otherwise you won't be able to fully use the app.
For example you can claim multiple periods, becoming a member requires multiple actions

Reply:
fixed

Reply:
https://github.com/JoinSEEDS/seeds_light_wallet/pull/341/commits/c3a9814ee3a2ce795d607f6012b63847ef2f0955